manageiq-operator/pkg/controller/manageiq/manageiq_controller.go

```diff
@@ -231,31 +231,31 @@ func (r *ReconcileManageIQ) generateMemcachedResources(cr *miqv1alpha1.ManageIQ)
 }
 
 func (r *ReconcileManageIQ) generatePostgresqlResources(cr *miqv1alpha1.ManageIQ) error {
-	postgresqlSecret := miqtool.DefaultPostgresqlSecret(cr)
-	if err := r.createk8sResIfNotExist(cr, postgresqlSecret, &corev1.Secret{}); err != nil {
+	secret := miqtool.DefaultPostgresqlSecret(cr)
+	if err := r.createk8sResIfNotExist(cr, secret, &corev1.Secret{}); err != nil {
 		return err
 	}
 
-	postgresqlConfigsConfigMap := miqtool.NewPostgresqlConfigsConfigMap(cr)
-	if err := r.createk8sResIfNotExist(cr, postgresqlConfigsConfigMap, &corev1.ConfigMap{}); err != nil {
+	configMap, configMapMutateF := miqtool.NewPostgresqlConfigsConfigMap(cr)
+	if _, err := controllerutil.CreateOrUpdate(context.TODO(), r.client, configMap, configMapMutateF); err != nil {
```

Comment:
Can you log the result of the call [as we do for memcached](https://github.com/ManageIQ/manageiq-pods/blob/0e3e06e2ac81923b8fb27064b1eef40b599d1d54/manageiq-operator/pkg/controller/manageiq/manageiq_controller.go#L217-L228)?

(Applies to all of the `CreateOrUpdate` calls in this function)